src/opts.rs

```diff
@@ -29,6 +29,10 @@ pub struct Opts {
     #[arg(short = 'H', long)]
     pub hidden: bool,
 
+    /// Silences output
+    #[arg(short = 'Q', long)]
```

Comment:
Short flag should be a lowercase 'q'.
```suggestion
    #[arg(short = 'q', long)]
```

Reply:
you are right! :-)